client/asset/eth/eth_test.go

```diff
@@ -134,10 +135,32 @@ func (n *testNode) initiate(ctx context.Context, contracts []*asset.Contract, ma
 		Nonce: n.nonce,
 	}), nil
 }
-func (n *testNode) redeem(opts *bind.TransactOpts, redemptions []*asset.Redemption, contractVer uint32) (*types.Transaction, error) {
-	return nil, nil
+
+func (n *testNode) redeem(ctx context.Context, redemptions []*asset.Redemption, maxFeeRate uint64, contractVer uint32) (*types.Transaction, error) {
+	if n.redeemErr != nil {
+		return nil, n.redeemErr
+	}
+	/*baseTx := &types.DynamicFeeTx{
```

Comment:
Did you want to leave these comments?

Reply:
No.. I'll take them out of initiate also.